magpie-run

```diff
@@ -2396,6 +2562,23 @@ then
     fi
 fi
 
+if [ "${ZEPPELIN_SETUP}" == "yes" ] && [ "${zeppelin_should_be_torndown}" == "1" ]
+then
+    if [ ${ZEPPELIN_MODE} != "setuponly" ]
+    then
+	cd ${ZEPPELIN_HOME}
+
+	# Make variables unspecified for shutdown
+	#Magpie_make_all_local_dirs_unspecified
+
```

Comment:
We only start one daemon on the master node. Thus there is no need to change the dirs. I have removed these to avoid confusion.